Libraries/Utilities/ReactNativeTestTools.js

```diff
@@ -78,9 +72,6 @@ function byTestID(testID: string): Predicate {
 
 function byTextMatching(regex: RegExp): Predicate {
   return withMessage(
-    /* $FlowFixMe(>=0.125.1 site=react_native_fb) This comment suppresses an
-     * error found when Flow v0.125.1 was deployed. To see the error, delete
-     * this comment and run Flow. */
     node => node.props && regex.exec(node.props.children),
```

Comment:
and this to 
```
node => node.props != null && regex.exec(node.props.children) !== null,
```

Reply:
@lunaleaps The `!=` in first check is a typo?